code/modules/projectiles/guns/projectile/shotgun/type21.dm

```diff
@@ -0,0 +1,47 @@
+/obj/item/gun/projectile/shotgun/type_21
+	name = "OS Type 21 SG \"Yaoguai\"" //Demon
+	desc = "an early model shotgun used by onestar. its compact with a sawn down barrel and no stock. and designed for police use in megacities its recoil kicks badly but the stopping power is worth it. its fed from drum magazines"
+	icon = 'icons/obj/guns/projectile/os/type_21.dmi'
+	icon_state = "type_21"
+	item_state = "type_21"
+	w_class = ITEM_SIZE_BULKY
+	force = WEAPON_FORCE_PAINFUL
+	slot_flags = SLOT_BACK|SLOT_BELT|SLOT_HOLSTER //isnt fitting in holsters for some reason
+	origin_tech = list(TECH_COMBAT = 6, TECH_MATERIAL = 3)
+	caliber = CAL_SHOTGUN
+	load_method = MAGAZINE
+	mag_well = MAG_WELL_RIFLE
+	magazine_type = /obj/item/ammo_magazine/m12
+	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_PLASTIC = 10)
+	price_tag = 2000
+	spawn_blacklisted = TRUE //until onestar rework
+	fire_sound = 'sound/weapons/guns/fire/shotgunp_fire.ogg'
+	unload_sound = 'sound/weapons/guns/interact/ltrifle_magout.ogg'
+	reload_sound = 'sound/weapons/guns/interact/ltrifle_magin.ogg'
+	cocked_sound = 'sound/weapons/guns/interact/hpistol_cock.ogg'
+	damage_multiplier = 1.0
+	penetration_multiplier = 0.5 // this is not babies first gun. It's a Serb-level weapon.
```

Comment:
Sir this "One Star Shotgun" is just the Bojevic.

Reply:
stats are somewhat changed and its literally the only other magfed semi auto shotgun in the game

Reply:
there removed the comment